### DISCUSSION TITLE
Users list not loading

### DISCUSSION THREAD

**Discussion by mtna-matt-wolfe** [nilsteampassnet/TeamPass#4004] [General]
I am unable to load my user list in 3.1.0.11. when i click on users nothing is listed and there is a loading message in the bottom right of the screen.  I recently added a CA certificate for teampass in apache and the user list was working prior to this change.

Any help would be greatly appreciated.

**Comment by MohamedElshafie-97** [comment]
enter settings --> TeamPass location (URL) --> put https to your website
